wgpu/examples/water/main.rs

```diff
@@ -528,7 +528,11 @@ impl framework::Example for Example {
             // Fragment shader and output targets
             fragment: Some(wgpu::FragmentState {
                 module: &water_module,
-                entry_point: "fs_main",
+                entry_point: if cfg!(feature = "webgl") {
```

Comment:
The split here is about RODS not webgl. We should be checking for the RODS downlevel flag, not based on platform: https://docs.rs/wgpu/0.12.0/wgpu/struct.DownlevelFlags.html#associatedconstant.READ_ONLY_DEPTH_STENCIL

Reply:
Got it, I'll update it.

Reply:
Fixed!